lms/static/js/spec/financial-assistance/financial_assistance_form_view_spec.js

```diff
@@ -189,6 +197,21 @@ define([
                 failedSubmission();
                 successfulSubmission();
             });
+
+            describe('when no country', function(){
+                beforeEach(function() {
+                    context.user_details.country = "";
+
+                    view = new FinancialAssistanceFormView({
+                        el: '.financial-assistance-wrapper',
+                        context: context
+                    });
+                });
+
+                it('renders invalid country', function() {
+                    invalidCountry();
+                });
```

Comment:
It there also a test checking the opposite (i.e. when there IS a country that the submit button is not disabled and no errors show on page load?